src/mongoose_cluster_id.erl

```diff
@@ -47,6 +50,48 @@ start() ->
             {error, conflict}
     end.
 
+
+%% If RDBMS is available before CETS - it is enough for us to continue
```

Comment:
1. here we want to ensure that we read existing cluster ID - so we want to access our another peer (CETS node or RDBMS). If CETS is not ready, and we don't wait for RDBMS - we will just generate a new cluster ID, which we do not want.
2. It speeds up starting time. We block the whole MIM initialization here, so if we can just unblock after RDBMS returns - it could be faster.